typescript/sdk/src/providers/SmartProvider/SmartProvider.ts

```diff
@@ -424,15 +435,20 @@ export class HyperlaneSmartProvider
     const rpcServerError = errors.find((e) =>
       RPC_SERVER_ERRORS.includes(e.code),
     );
-    const timedOutError = errors.find(
-      (e) => e.status === ProviderStatus.Timeout,
```

Comment:
Are you sure you'd like to remove this? The purpose was to identify when a failure was due to SmartProvider's timeout handling vs an underlying error

Reply:
Very good point. That was my original intention, but got confused